pycbc/workflow/datafind.py

```diff
@@ -530,7 +533,6 @@ def setup_datafind_runtime_cache_single_call_perifo(cp, scienceSegs, outputDir,
                 cache, cache_file = run_datafind_instance(
```

Comment:
Same question for this try/except

Reply:
Same answer as above, expect this code is actually used, and is important. I'm hesitant to remove it at this stage in case it is needed. I'm not seeing such failures *now*, but who knows what will happen at other times.